### DISCUSSION TITLE
Date picker

### DISCUSSION THREAD

**Discussion by hoannguyen02** [Shopify/function-examples#389] [Feedback: General Improvements]
Hi everyone,

We have an issue related date functionality and already opened it [here](https://github.com/Shopify/ui-extensions/issues/1217). 

Best regards,
Hoan

**Comment by nickwesselman** [comment]
Hi @hoannguyen02 -- I am going to close this since it's related to UI extensions and not Functions.

**Reply by hoannguyen02** [reply]
Hi @nickwesselman thanks for letting me know. Honestly I don't really know where to post and who can help me on this.
